### PR TITLE
Updated slack notification and performed code refactoring

### PR DESCRIPTION
**Pull request by yashashreesuresh**
This commit:
- Updates slack notification to report the node failures
- Refactors the code to improve the code readability

### CODE REVIEW

**Comment by yashashreesuresh** [comment]
Please have a look @chaitanyaenr

File: start_cerberus.py [hunk 1]
```diff
@@ -69,27 +69,16 @@ def main(cfg):
...
-
-        # Remove 'inspect_data' directory if it exists.
-        # 'inspect_data' directory is used to collect
-        # logs, events and metrics of the failed component
-        if os.path.isdir("inspect_data/"):
-            logging.info("Deleting existing inspect_data directory")
-            runcommand.invoke("rm -R inspect_data")
+            slack_integration = slackcli.initialize_slack_client()
+
+        if inspect_components:
```
Comment (chaitanyaenr) [inline start_cerberus.py:74]: Maybe move the delete operation to line 144 where the data collection happens when inspect_components is set to true? Thoughts? Also, let's add a log message when inspection is enabled.

File: start_cerberus.py [hunk 2]
```diff
@@ -69,27 +69,16 @@ def main(cfg):
 
         # Create slack WebCleint when slack intergation has been enabled
         if slack_integration:
```
Comment (chaitanyaenr) [inline start_cerberus.py:71]: Let's add a log message when the slack integration is enabled.

File: start_cerberus.py (hunk 1 above)
Comment (yashashreesuresh) [reply start_cerberus.py:74]: Delete operation needs to be outside the while loop as the directory is deleted only once when Cerberus starts monitoring.

Comment (chaitanyaenr) [reply start_cerberus.py:74]: Makes sense. Thanks.

**Pull request merged**